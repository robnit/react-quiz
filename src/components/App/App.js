import React, { Fragment } from 'react'
import Books from 'components/Books/Books'

const App = () => (
  <Fragment>
    <h1>REACT QUIZG</h1>
  <Books/>
  </Fragment>
)

export default App

/* -------------------------------------------------------- *
 *                                                          *
 * -------------------------------------------------------- */