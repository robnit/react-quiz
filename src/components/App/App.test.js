import React from 'react'
import App from 'components/App/App'
import { mount } from 'enzyme' 

describe('app', () => {
  it('should match snapshots', () => {
    const wrapper = mount(<App/>)
   
    expect(wrapper).toMatchSnapshot()
  })
})
