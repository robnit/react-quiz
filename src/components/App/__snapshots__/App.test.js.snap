// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should match snapshots 1`] = `
<App>
  <h1>
    REACT QUIZG
  </h1>
  <Books>
    <AddBooks>
      <h1>
        AddBooks
      </h1>
    </AddBooks>
    <Book>
      <h1>
        BOOKS
      </h1>
    </Book>
  </Books>
</App>
`;
